src/main/java/com/jalasoft/jfc/controller/ImageConverterController.java

```diff
@@ -82,7 +87,9 @@
      * @param request contains client request data.
      * @return Response it mean the result of the conversion.
      */
-    @PostMapping()
+    @PostMapping("/imageConverter")
+    @ApiOperation(value = "Image specifications", notes = "provide values for converting Image file to other one",
```

Comment:
Use magick number here please

Reply:
in the comment ?

Reply:
No, in code line 91 "provide ...."

Reply:
If it is only a note, you can fix it in other feature